test/planning/test_task_objects.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+
+"""Unit tests for task action and plan objects"""
+
+import pytest
+
+from pyrobosim.planning.actions import TaskAction, TaskPlan
+from pyrobosim.utils.motion import Path
+from pyrobosim.utils.pose import Pose
+
+
+def test_task_action_default_args():
```

Comment:
Some comments saying in high level terms what everything is testing would help someone looking at these in the future